packages/connect/src/api/common/MiscFees.ts

```diff
@@ -56,7 +60,7 @@ export class MiscFeeLevels {
 
     updateCustomFee(feePerUnit: string) {
```

Comment:
As `EthereumFeeLevels.updateEthereumCustomFee` was removed (probably unused), could this `MiscFeeLevels.updateCustomFee` be removed as well?